AuthenticationDemo/ExampleSessionService.swift

```diff
@@ -125,3 +141,65 @@ public final class ExampleSessionService: SessionServiceType {
     }
     
 }
+
+fileprivate extension ExampleSessionService {
+
+    fileprivate func bindPersistance() {
```

Comment:
Should this be implemented in every session service? Cannot provide a default implementation?